spec/controllers/project_folders_controller_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ProjectFoldersController, type: :controller do
+  login_user
+  render_views
+
+  let!(:user) { User.first }
+  let!(:team) { create :team, created_by: user }
+  let!(:user_team) { create :user_team, team: team, user: user, role: :admin }
+
+  let!(:project_folder_1) do
```

Comment:
Try to avoid using `let!`. 
When we add additional tests, all those objects will be instanced before each test block. Maybe use before block inside post?